mmv1/products/cloudfunctions2/terraform.yaml

```diff
@@ -118,8 +112,6 @@ overrides: !ruby/object:Overrides::ResourceOverrides
       eventTrigger.eventFilters: !ruby/object:Overrides::Terraform::PropertyOverride
         is_set: true
     iam_policy: !ruby/object:Api::Resource::IamPolicy
-      min_version: beta
       parent_resource_attribute: 'cloud_function'
       method_name_separator: ':'
       import_format: ["projects/{{project}}/locations/{{location}}/functions/{{cloud_function}}", "{{cloud_function}}"]
-      example_config_body: 'templates/terraform/iam/example_config_body/cloudfunctions2_function.tf.erb'
```

Comment:
if this override is not needed, should this be moved to api.yaml?

Reply:
@c2thorn

Reply:
No worries, this was related to IAM documentation, but I think it is better without it. Thanks for cleaning up.